ivadomed/main.py

```diff
@@ -211,14 +203,16 @@ def cmd_train(context):
     in_channel = 1
 
     if context['multichannel']:
-        in_channel = len(context['multichannel'])
+        in_channel = len(context['contrast_train_validation'])
```

Comment:
When `contrast_train_validation` was introduced in the config files, it was used in the following context: Train a Unet on  `contrast_train_validation` and inference on `contrast_testing`.
What was the reason to get ride of `multichannel` param?